code_strategy.md

```diff
@@ -0,0 +1,112 @@
+We are biomedical data scientists who use code to answer biological hypotheses and explore biological data. It is therefore important to define how we use, write, and share code. 
```

Comment:
```suggestion
We are biomedical data scientists who use code to answer biological hypotheses and explore biological data.
It is therefore important to define how we use, write, and share code. 
```

Please go through all `.md` files and make sure they are 1 sentence per line

Reply:
Done in https://github.com/WayScience/playbooks/pull/1/commits/1a77a3e27ee55d497b90e3bf26f64485a473e9ca